executor/insert_common.go

```diff
@@ -1126,6 +1128,13 @@ func (e *InsertValues) batchCheckAndInsert(ctx context.Context, rows [][]types.D
 			defer snapshot.SetOption(kv.CollectRuntimeStats, nil)
 		}
 	}
+	sc := e.ctx.GetSessionVars().StmtCtx
+	for _, fkc := range e.fkChecks {
```

Comment:
any test case about insert into select from and load data？

Reply:
Already add test for `insert  into select`.

Currently `load data` doesn't do foreign check, so can not add test for `load data`.